R10 Upload changes/fixes

jseppi opened the pull request:
- Add R10 spreadsheet example to R10 Upload Step 1: Closes https://github.com/18F/calc/issues/588
- Fix R10 upload page heading: Closes https://github.com/18F/calc/issues/849
- Minor R10 step template rearrangements

<img width="769" alt="screen shot 2016-10-03 at 3 28 01 pm" src="https://cloud.githubusercontent.com/assets/697848/19054786/2f63c12a-897e-11e6-8b04-e0d750f0c315.png">

jseppi commented on frontend/source/sass/components/_exceltables.scss:
```diff
@@ -32,7 +32,8 @@ table.excel {
 
-table.excel.excel-schedule-70 {
+table.excel.excel-schedule-70,
+table.excel.excel-region-10 {
```
Not too sure why the `.excel-schedule-70` rules are not just in the `.excel` class. @toolness?

toolness replied:
Modularity, basically--I wanted the `.excel` class to be like a "vanilla" excel spreadsheet that we could then build separate ones off of, depending on the styling of different price lists or CSVs.  Explained in the [style guide](https://calc-dev.apps.cloud.gov/styleguide/#excel-tables)!

That said, the schedule 70 one was intentionally designed to look like the S70 PPL table.  Having a region 10 one might make it seem (to devs) like that is how "the official region 10 bulk upload" sheet looks... If there *is* no such official spreadsheet, it might make sense to call the new class `table.excel.excel-generic` or something, to indicate that it's not meant to reflect a particular "official" style.

jseppi replied:
cool, I like that idea 👍

jseppi commented:
ready for review, @toolness

toolness commented on data_capture/templates/data_capture/price_list/upload_examples/region_10.html:
```diff
@@ -0,0 +1,30 @@
+  </p>
+  <p class="dev-note">
+    <strong>NOTE:</strong> If you have the Debug Toolbar enabled, you will not be able to import large datasets.
+    You can disable it (and other dev/debug affordances) by setting <code>HIDE_DEBUG_UI=yup</code> in your local <code>.env</code> file.
```
Is this still true, or was this made irrelevant once we moved to the async way of doing things?

jseppi replied:
Just checked, and it indeed works now that the processing is async

toolness commented:
Nice, this looks good to me!

toolness commented:
As an aside, I noticed the first paragraph on step 1 reads:

> Upload an exported Region 10 pricing database spreadsheet into CALC. **This will delete and replace all of the existing Region 10 data in CALC!**

That first sentence reads odd to me... I'm not sure how to describe it, but It sounds like the name of a title or heading rather than the first sentence of a paragraph.  If it doesn't sound weird to you then I'm probably just imagining things, but otherwise maybe we should file a separate issue about it?

jseppi commented:
Ok, I
- changed to just using the base `table.excel` styling,
- removed the irrelevant note about disabling djdt
- changed the intro text to be a more good sentence, and simplified the part about replacing existing r10 data.
